app/routes/bdb/components/BdbDetail.js

```diff
@@ -404,6 +421,7 @@ export default class BdbDetail extends Component<Props, State> {
                     <th>Når</th>
                     <th>Hvor</th>
                     <th>Hva</th>
+                    <th />
```

Comment:
Would it make sense to write 'spørreundersøkelser' in order to clarify explicitly where the link goes?

Reply:
Concluded that 'spørreundersøkelse' became a bit much as a title for a simple icon. Instead added a tooltip hoovering effect. 
![Skjermbilde 2019-11-08 kl  17 17 13](https://user-images.githubusercontent.com/43182013/68493070-0a316200-024c-11ea-84fc-5f9957d6d2fc.png)